content/synthetics/uptime_check.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Uptime Check
+kind: documentation
+beta: true
+description: Simulate and monitor HTTP requests from specific locations
+further_reading:
+- link: "https://www.datadoghq.com/blog/introducing-synthetic-monitoring/"
+  tag: "Blog"
+  text: "Introducing Datadog Synthetics"
+- link: "synthetics/"
+  tag: "Documentation"
+  text: "Manage your checks"
+- link: "synthetics/browser_check"
+  tag: "Documentation"
+  text: "Configure a Browser Check"
+---
+
+## Overview
+
+Uptime checks are HTTP requests (GET, POST, PUT, etc.) executed by Datadog to your web properties or application endpoints at configurable periodic intervals from multiple locations around the world. These checks verify not only that your applications are up and responding to requests, but that any conditions you define are met, such as response time, HTTP response code, and header or body contents.
+
+## Configuration
+### Request
+
+Define the request you want to be executed by Datadog:
+
+{{< img src="synthetics/uptime_check/uptime_check_make_request.png" alt="Uptime check make request" responsive="true" style="width:80%;">}}
+
+1. Choose the `Method` and `URL` to query. Available `Method` are:
```

Comment:
Available methods - plural. But it's also a field name. Hmm. I'm not sure what to do there. 🤷‍♂️

Reply:
let's keep the parameter name.

Reply:
Available values for `Method` are: